### DISCUSSION TITLE
Carbon Atoms Symbols

### DISCUSSION THREAD

**Discussion by nick-charistos** [jsme-editor/jsme-editor.github.io#44] [Q&A]
Hi JSME community

How can JSME show the carbon atoms symbols "C" in a structure?  

I have found a workaround using [C] in SMILES or .mol and it worked. 

But is there an option in JSME to always show the carbon atoms symbols? 

Thanks

Nickolas

**Comment by mykcci** [comment]
Hi Nickolas,

Sorry, I don't have an answer to your question.  There's nothing in the JSME documentation, that I can find, about displaying C symbols.  That would be a useful feature.

Your workaround is great, thank you very much.  It enabled me to use JSME in an education site in a role that it couldn't otherwise perform.

Many thanks,
Mike
